### DISCUSSION TITLE
I am facing issue with arrow and rounded in pagination tailwindcss with RTL

### DISCUSSION THREAD

**Discussion by allaghi** [gilbitron/laravel-vue-pagination#175] [Help & support]
Hello everyone,
Have a nice day,
I am facing issue when I use it with RTL and you can see my issue with attachment image.
as you can see I am trying to change rounded corners. so how can I do it.
Please if you have any solution , Shoot 

![screenshot-localhost_8000-2023 09 20-10_19_33](https://github.com/gilbitron/laravel-vue-pagination/assets/4201457/041d770b-b2ba-418e-a606-e9d6b15cabc1)

Thanks in advance.

**Comment by gilbitron** [comment]
This is a bug. I've created an issue for it in #176
